integration_tests/__tests__/require_all_modules.test.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+'use strict';
+
+const runJest = require('../runJest');
+
+it('Makes sure that no native module makes Jest crash', () => {
+  const result = runJest.json('require-all-modules').json;
+
+  if (!result.success) {
+    console.warn(result);
```

Comment:
for debugging? should it be removed?

Reply:
I added it so that if a module throws when required, we can know who was based on the console 🙂